### ISSUE TITLE
Definition lists for show page components

### ISSUE THREAD

**Issue by nikdragovic** [emory-libraries/dlp-lux#656]
## Acceptance Criteria
- [ ] For “this item is part of / this item contains / view items in this digital collection" section, ensure that the definition list is constructed correctly so that screen readers don’t give confusing output.
- [ ] For Access and Copyright sections, ensure that the definition list is constructed correctly so that screen readers don’t give confusing output.

**Comment by bwatson78** [comment]
@nikdragovic 
Recommendations: 

- All card divs that have classes that contain the card title hyphenated ("This Item Is Part Of" -> class="is-part-of") should also have a title attribute that repeats the same card title.
- Each dl directly inside the div with the class "card-body" should have a descriptive title.
- The "Tools" card lacks the class "card-body", probably because of styling reasons. I'd recommend the title attribute go on the div with the classes "card mb-2 rounded-0". When I researched the accessibility of uls/lis, I found that there shouldn't be any issue, unless it's a list of menu options (I think this counts as a menu.) We may consider adding a title to the ul and each li to fully describe what their options are here.

**Comment by bwatson78** [comment]
![Screen Shot 2020-07-24 at 7.29.16 AM.png](https://images.zenhubusercontent.com/5e6f6c00c3241244877b4aea/2e7c7120-ee58-4f31-a610-ddf69dd05873)
![Screen Shot 2020-07-24 at 7.28.34 AM.png](https://images.zenhubusercontent.com/5e6f6c00c3241244877b4aea/d0c50288-b6ed-4d29-a26c-7616b133679b)

**Comment by bwatson78** [comment]
PR made: https://github.com/emory-libraries/dlp-lux/pull/666
